src/wpool_process_sup.erl

```diff
@@ -55,3 +56,13 @@ init({Name, Options}) ->
       , [Worker]
       } || I <- lists:seq(1, Workers)],
   {ok, {Strategy, WorkerSpecs}}.
+
+maybe_add_event_handler(Options) ->
+  case proplists:get_value(event_manager, Options, undefined) of
+    undefined ->
+      ok;
+    EventMgr ->
+      [gen_event:add_handler(EventMgr, {wpool_process_callbacks, Module}, Module)
```

Comment:
Since we don't care about the result of this list comprehension, why not using `lists:foreach` instead?